### PR TITLE
Fix GitHub brand

### PR DESCRIPTION
**Pull request by chrisnojima**
Base code from: https://github.com/keybase/client/pull/5755
Additionally fixes flow issues on the native side. Making my own PR so I can test vs CI

@montogeek
@keybase/react-hackers

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits 287734c3b5cc...acc05f80a078 introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Edit_Avatar-has_avatar** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1487781667768-287734c3b5cc-acc05f80a078-linux/Edit_Avatar-has_avatar-287734c3b5cc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1487781667768-287734c3b5cc-acc05f80a078-linux/Edit_Avatar-has_avatar-acc05f80a078.png)  
   Edit_Avatar-has_avatar comparison
 * Changed: **Edit_Avatar-does_not_have_avatar** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1487781667768-287734c3b5cc-acc05f80a078-linux/Edit_Avatar-does_not_have_avatar-287734c3b5cc.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1487781667768-287734c3b5cc-acc05f80a078-linux/Edit_Avatar-does_not_have_avatar-acc05f80a078.png)  
   Edit_Avatar-does_not_have_avatar comparison
</summary>

File: shared/actions/search.js [hunk 1]
```diff
@@ -149,7 +149,7 @@ function search (term: string, maybePlatform: ?SearchPlatforms) : TypedAsyncActi
     const service = {
       'Coinbase': 'coinbase',
       'Facebook': 'facebook',
-      'Github': 'github',
+      'GitHub': 'github',
       'Hackernews': 'hackernews',
```
Comment (hugovk) [inline shared/actions/search.js]: Whilst you're at it, you could fix "Hackernews" -> "Hacker News" and maybe "Pgp" -> "PGP" (here and elsewhere).

File: shared/dev/log-send/render.native.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ class LogSendRender extends Component<void, Props, State> {
           </Text>
           {this.state.copiedToClipboard && <Text type='Body'>Copied to clipboard!</Text>}
 
-          <Text type='Body'>Send us the log id along with a description of what's going on in this Github issue:</Text>
+          <Text type='Body'>Send us the log id along with a description of what's going on in this GitHub issue:</Text>
           <Button type='Primary' label='File a Github issue:' onClick={onSubmitIssue} />
```
Comment (hugovk) [inline shared/dev/log-send/render.native.js]: Missed a "Github" in the label.

**Comment by chromakode** [comment]
@chrisnojima bump!

**Comment by chrisnojima** [comment]
Just a quick update. I will get this in but it's behind us getting out mobile app out. We're also going to be updating the search to a very exciting new design right after this so we'll definitely roll this in

**Comment by chrisnojima** [comment]
ok so good news and bad news. Good news is we're entirely revamping search right now with the correct names (Thanks for the feedback!). Bad news this pr isn't going to move forward.

**Pull request closed without merging**